logout.php

```diff
@@ -1,15 +1,15 @@
-<?php
-  require_once('views/head.php');
-?>
-  <title>Comicdb</title>
+<?php require_once('views/head.php'); ?>
+  <title>Logout :: comicDB</title>
 </head>
 <body>
-  <?php include 'views/header.php'; ?>
-  <div class="mdl-layout">
-    <div class="mdl-grid">
-      Logged out.
-      <br/>
-      <a href="<?php echo filter_input ( INPUT_GET, 'return' ); ?>">Go Back</a>
+  <?php include 'views/header.php';?>
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-12">
+        Logged out.
+        <br/>
+        <a href="<?php echo filter_input ( INPUT_GET, 'return' ); ?>">Go Back</a>
+     </div>
     </div>
   </div>
   <?php include 'views/footer.php';?>
```

Comment:
I'd like to kill this entire page, and just send the user back to the index with a little animated drop down message bar that disappears after 5 seconds. A lot of the notifications (Adding series/issue confirmation/edit saved) will be done this way in my next PR.